templates/test/ci/patches/maxpods-aks.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
```

Comment:
sorry if I'm being annoying but... can you please move this to a `patches` directory under https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/main/templates/test/ci/prow-aks-multi-tenancy? that's the current pattern for other flavors, eg. https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/main/templates/test/ci/prow-custom-vnet/patches, it's just tags that's an exception because it's shared between all the flavors